Doc/library/dis.rst

```diff
@@ -41,6 +41,10 @@ the following command can be used to display the disassembly of
 
 (The "2" is a line number).
 
+:mod:`dis` an also be invoked as a script to disassemble other scripts. For example::
```

Comment:
take care the page width  (79 chars, if I'm not wrong)

Reply:
Can you add more detail about the args (infile)?

Reply:
I mean, it seems pretty straightforward. What more to explain other than feed it a python script?